src/api/v1/serializers/ingredients-serializer.js

```diff
@@ -34,4 +45,27 @@ const serializeIngredients = (rawIngredients, query) => {
   ).serialize(rawIngredients);
 };
 
-export { serializeIngredients };
+/**
+ * Serializes a single ingredient
+ * @param {object} rawIngredient
```

Comment:
Blank line between description and @param.

Reply:
I promise my next PR will have the right spacing in all the jsdoc comments 😢